Corona-Warn-App/src/main/java/de/rki/coronawarnapp/profile/model/Profile.kt

```diff
@@ -0,0 +1,24 @@
+package de.rki.coronawarnapp.profile.model
+
+import org.joda.time.LocalDate
+
+data class Profile(
+    val id: String? = null,
```

Comment:
is it really `optional` if we have a profile all fields could be optional ,but we still have a profile

Reply:
This is for new profiles, the id will be assigned on insert